tests/get_table_tests.cpp

```diff
@@ -116,6 +116,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE( get_scope_test, TESTER_T, backing_store_ts) { try
       BOOST_REQUIRE_EQUAL(name("initc"_n), result.rows[1].scope);
    }
 
+   param.lower_bound = "initb";
+   param.upper_bound = "initd";
```

Comment:
I missed that there was a scope with "initd", so you need to set upper_bound = "inite" (or past the last scope, so you could do param.upper_bound = "z"; which should guarantee that is past the last scope, since I doubt it has any scope values that start with "z".

Reply:
Updated